packages/pilot/src/components/Operations/operationsTableColumns.js

```diff
@@ -77,24 +81,76 @@ const renderTargetOrSource = (type, net, targetId, sourceId, labels) => {
   return null
 }
 
+const greaterThanZero = gt(__, 0)
+
+const isFailedTransfer = allPass([
+  either(
+    propEq('type', 'credito_em_conta'),
+    propEq('type', 'transfer')
+  ),
+  propEq('status', 'failed'),
+  pipe(
+    prop('amount'),
+    greaterThanZero
+  ),
+])
+
+const getOutgoingAmount = pipe(
+  head,
+  prop('amount')
+)
+
+const renderCreditTransferStatus = (
+  type, status, outgoing, transferId, labels
```

Comment:
Já que a linha está sendo quebrada pq não colocar assim:
```suggestion
  type,
  status,
  outgoing, 
  transferId,
  labels
```

Reply:
Corrigido!